front-end/src/screens/Home/HomeContent.tsx

```diff
@@ -65,40 +65,39 @@ const HomeContent = ({ className, data }: Props) => {
 
 export default styled(HomeContent)`
 
-	@media only screen and (max-width: 576px) {
+	@media only screen and (max-width: 768px) {
 		h3 {
-			margin-left: 1.5rem!important;
+			margin-left: 0rem;
 		}
 
 		.mainButtonContainer {
-			align-items: stretch!important;
-			margin: 1rem!important;
-
-			.newPostButton {
-				padding: 0.8rem 1rem;
-				border-radius: 0.5rem;
-			}
+			margin: 0rem;
 		}
 	}
 
-	@media only screen and (max-width: 768px) {
-		h3 {
-			margin-left: 0rem;
+	@media only screen and (max-width: 991px) and (min-width: 768px) {
+		.ui[class*="tablet reversed"].grid {
+			flex-direction: column-reverse;
 		}
 
 		.mainButtonContainer {
-			margin: 1rem 0;
 			margin-top: 1rem!important;
 		}
 	}
 
-	@media only screen and (max-width: 991px) and (min-width: 768px) {
-		.ui[class*="tablet reversed"].grid {
-			flex-direction: column-reverse;
+	@media only screen and (max-width: 576px) {
+		h3 {
+			margin-left: 1.5rem!important;
 		}
 
 		.mainButtonContainer {
-			margin-top: 1rem!important;
+			align-items: stretch!important;
+			margin: 1rem!important;
+
+			.newPostButton {
+				padding: 0.8rem 1rem;
+				border-radius: 0.5rem;
+			}
 		}
 	}
 
```

Comment:
While we're at it, the 
```css
	.mainButtonContainer {
...
		margin-top: 4rem;
```

bellow causes a super large top margin when being at e 660px